__mocks__/locationMock.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  location: {
+    href: 'http://localhost:3000/openmrs-standalone/owa/openmrs-owa-orderentry/index.html#',
```

Comment:
Do you want the above to be standalone?

Reply:
This is for test, it allows me to mock the url. This is necessary in cases where we run the automated tests on continuous integration service like travis where the window.location object is not available.